README.md

```diff
@@ -66,7 +66,7 @@ You are now ready to login to Aras and check out the examples for customizing no
 
 ### Change the Background Color
 
-![Custom Background Color](./Screenshots/bg-color.gif)
+![Custom Background Color](./Screenshots/color_bkg-split.png)
```

Comment:
The split images are confusing. Either display two whole images or use a gif to toggle between the two. Otherwise, it just looks like there's something wrong with the rendering of the image.